resources/assets/js/app/controllers/UserController.js

```diff
@@ -0,0 +1,20 @@
+var Marionette = require('backbone.marionette');
+var app = require('../instances/appInstance');
+
+module.exports = Marionette.Object.extend({
+
+    index: function () {
+        var Users = require('../collections/userCollection');
+        var UsersView = require('../views/users/userCollection');
+        var users = new Users();
+
+        users.fetch({
+            success: function (items) {
+                app.RootView.content.show(new UsersView({
```

Comment:
And use getRegion to get region instead of direct usage, in future versions this approach will be removed